model/src/test/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModelSpec.scala

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.dsde.rawls.model
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.scalatest.{FreeSpec, Matchers}
+
+class WorkspaceModelSpec extends FreeSpec with Matchers {
+
+  "MethodRepoMethod" - {
+
+    val goodMethod = MethodRepoMethod("test-namespace", "test-name", 555)
+    val badMethod1 = MethodRepoMethod("a", "", 1)
+    val badMethod2 = MethodRepoMethod(null, "b", 1)
```

Comment:
I don't think you need the `null` test because it's not possible to achieve given the code.